eglot-grammarly.el

```diff
@@ -48,10 +48,17 @@ Link: https://github.com/znck/grammarly"
 
 (defun eglot-grammarly--server-command ()
   "Generate startup command for Grammarly language server."
-  (list "@emacs-grammarly/grammarly-languageserver" "--stdio"))
+  (list 'eglot-grammarly-server "grammarly-languageserver" "--stdio"))
```

Comment:
Can you change this back to `@emacs-grammarly/grammarly-languageserver`? AFAIK, the author of grammarly-languageserver doesn't use other editors so it's mostly like going to be unstable.

Reply:
Quick question about this. After I install the server using npm as mentioned in the readme, I cannot find a binary by running "@emacs-grammarly/grammarly-languageserver --stdio". I can run it with "grammarly-languageserver --stdio"
Did I miss anything?

Reply:
Oh wait, please ignore my previous comment. I thought the command was for installation. My bad!

`grammarly-languageserver` should be good. 😅